sdk/compute/Azure.ResourceManager.Compute/src/Customize/Models/UserInitiatedRedeploy.cs

```diff
@@ -0,0 +1,23 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// <auto-generated/>
+
+#nullable disable
+
+using System;
+using System.Collections.Generic;
+
+namespace Azure.ResourceManager.Compute.Models
+{
+    public partial class UserInitiatedRedeploy
+    {
+        internal UserInitiatedRedeploy(bool? automaticallyApprove, string dummyProperty, IDictionary<string, BinaryData> serializedAdditionalRawData)
+        {
+            AutomaticallyApprove = automaticallyApprove;
+            _serializedAdditionalRawData = serializedAdditionalRawData;
+        }
```

Comment:
I believe we do not have to have this ctor - just let it generate and assign values to dummy property, this ctor is internal, it is fine.